### PR TITLE
Added pagination for load photos

### PR DESCRIPTION
**Pull request by SvyatoslavArtymyvych**

### CODE REVIEW

**Review by varvarashcherbyna7 — APPROVED** [review]

File: app/views/photo_contest.py [hunk 1]
```diff
@@ -37,33 +38,37 @@ def load(photo_contest_uuid):
...
+                liked = "on"
+
+            photo_list.append(
+                [
+                    photo.file_name,
+                    photo.id,
+                    url_for("main.image", path=(photo.file_name)),
+                    liked,
+                ]
+            )
```
Comment (kipod) [inline app/views/photo_contest.py:66]:
```python
photo_list = [(photo.file_name, photo.id, url_for("main.image", path=(photo.file_name)), "on" if has_liked_photo(user=current_user, photo=photo) else "off") for photo in paginated_photos]
```

Comment (kipod) [reply app/views/photo_contest.py:66]: one line vs. 15 lines =)

File: app/views/photo_contest.py [hunk 2]
```diff
@@ -37,33 +38,37 @@ def load(photo_contest_uuid):
...
 
-        elif counter == posts:
+        if len(photo_list) == 0:
             res = make_response(jsonify({}), 200)
```
Comment (kipod) [inline app/views/photo_contest.py:69]:
```python
res = make_response(jsonify([]), 200)
```
returns array, not object

File: app/views/photo_contest.py [hunk 3]
```diff
@@ -3,8 +3,9 @@
 from flask import redirect, url_for, request, jsonify, make_response
 from flask_login import login_required, current_user
 from sqlalchemy import func
+from math import ceil
```
Comment (kipod) [inline app/views/photo_contest.py:6]: import of standard module must be before all others!

**Review by kipod — CHANGES_REQUESTED** [review]
thank you.
please see my comments

File: app/views/photo_contest.py (hunk 2 above)
Comment (varvarashcherbyna7) [reply app/views/photo_contest.py]: it seems to me that it should return object in which there is data with an array

**Pull request closed without merging**